lib/bankAccounts.js

```diff
@@ -205,6 +217,9 @@ export class BankAccounts {
    * @tag Bank accounts
    */
    async completeMicroDeposits(accountID, bankAccountID, amounts) {
+    checkString(accountID).or(Err.AccountID);
+    checkString(bankAccountID).or(Err.BankAccountID);
+
```

Comment:
Can you please add in a check for amounts too?

Reply:
You might need to update your branch to pull in the latest.  In the top of main there is a param description for amounts.  And we need to update it's description to be something like: ```Array of two positive integers, in cents, equal to the values of the micro-deposits sent to the bank account.```